src/strings.c

```diff
@@ -2888,7 +2888,11 @@ vim_vsnprintf_typval(
     if (parse_fmt_types(&ap_types, &num_posarg, fmt, tvs) == FAIL)
 	return 0;
 
-    va_copy(ap, ap_start);
+#ifdef va_copy
+    va_copy (ap, ap_start);
+#else
+    ap = ap_start;
+#endif
```

Comment:
Please also revert this change.

Reply:
I am sorry. Now I am confused. I have removed this part because of @k-takata  and @errael 's suggestion in https://github.com/vim/vim/pull/13553#discussion_r1407314149 .
Why should we revert this change? 
I fully agree that the proposed solution using #ifndef va_copy... is a superior solution.

Reply:
There were two `#ifdef va_copy` blocks in strings.c.
You reverted only one block (L2792), but L2891-L2895 still remains.

Reply:
Yes. Now I see.
I am sorry... I have missed the second block.
I have commited the change, please review.
Thank you